samples/snippets/csharp/new-in-6/classList.cs

```diff
@@ -0,0 +1,23 @@
+namespace NewStyle
+{
+    public class ClassList
+    {
+        public void method()
+        {
+            // <InitializeEnrollment>
+            var classList = new Enrollment()
+            {
+                new Student("Richard", "Feynman"),
```

Comment:
we can't use real people's names in examples. we have a database of fictitious names. let me know if you need help finding it.